docs/src/components/Layout/index.tsx

```diff
@@ -55,6 +72,49 @@ export default function Page({
             >
               {description}
             </Text>
+            <View className="docs-component-links">
+              {ariaPattern ? (
+                <Link
+                  className="docs-component-link"
+                  href={ariaPattern}
+                  isExternal
+                >
+                  <Icon
+                    ariaLabel=""
```

Comment:
These icon aria labels are empty because they do not provide any value to screenreaders as the link already has the applicable content.

Reply:
Is this the SVG equivalent to having empty alt tags on images?

Reply:
Correct. There might be a better way to do this? Working with icons for a bit now, there are 2 types: 
1. Without labels, like an 'x' icon meaning close. This could either have an ariaLabel or use `VisuallyHidden` with no ariaLabel
2. With labels, like in this case. The link already has an accessible label, provided by the text. The icon is purely graphical and would provide no benefit to screen readers (and might sound weird announcing a similar thing twice). 

I don't know if there is a better way to handle 2...